erddap/templates/private/dataset.deployment.xml

```diff
@@ -310,7 +310,7 @@
             <att name="ioos_category">Quality</att>
             <att name="long_name">{{dest_var_remaps.get(req_var, req_var).replace('_qc', '')}} Variable Quality Flag</att>
             {% if req_var not in qc_var_types['gen_qc'] %}
-            <att name="flag_values">0,1,2,3,4,5,6,7,8,9</att>
+            <att name="flag_values">0b, 1b, 2b, 3b, 4b, 5b, 6b, 7b, 8b, 9b</att>
```

Comment:
I'm mobile and can't test this in ERDDAP or the NCML syntax at the moment... but I believe this should be:

`<att name="flag_values" type="byte">0 1 2 3 4 5 6 7 8 9</att>`

I don't know where the `b` is coming from and spaces are used as the separator in NcML/NetCDF array attributes.

Can you add any tests for these templates? Maybe loading one into NetCDF Java with some fill data and testing the attributes?